xml/art-obs-beginners-guide.xml

```diff
@@ -930,68 +965,83 @@ Fri Aug 23 12:31:41 UTC 2017 - &exampleuser_mail;</screen>
+     macro:</para>
     <screen>%prep
 %setup -q -n %{name}-%{version}
 %patch0</screen>
    </step>
    <step>
     <para>
-     Add you patch file to the local repository and rebuild your package
-     with:
+     Add your patch file to the local repository and rebuild your package:
     </para>
     <screen>&prompt.user;<command>osc</command> add &gitproject;_main.diff
 &prompt.user;<command>osc</command> build openSUSE_Tumbleweed x86_64 *.spec</screen>
    </step>
    <step>
-    <para>If everything was successfull, commit your changes by adding
-     your commit message:
+    <para>
+     If everything was successful, commit your changes:
     </para>
     <screen>&prompt.user;<command>osc</command> commit</screen>
+    <para>
+     An editor will open, so you can specify and save a commit message.
+    </para>
    </step>
   </procedure>
   <para>
    If you have to deal with many patches, the tool
-   <systemitem>quilt</systemitem> could be useful.
+   <command>quilt</command> can be useful. For more information about
+   <command>quilt</command>, see
+   <link xlink:href="https://savannah.nongnu.org/projects/quilt"/>.
   </para>
+  <remark>
+   Homepage is admittedly not great, but did not want to link to Wikipedia. -
+   sknorr, 2017-08-16
+  </remark>
  </sect1>
 
  <sect1 xml:id="sec.obsbg.uc.branchprj">
-  <title>Branching a Project</title>
+  <title>Branching a Package</title>
+  <remark>
+   Is the OBS definition of branch really so narrow? (i.e. just for home
+   project copies?) - sknorr, 2017-08-16
+  </remark>
   <para>
-   This section describe how to collaborate with other projects.
+   This section describes how to collaborate between projects.
    A <firstterm baseform="Branch">branch</firstterm> in &obsa; terms is a
-   personal copy of another project that lives on your home project.
-   To be precise, all branches will be stored in a subproject of your
-   home and named as <uri>&obsbranch1;</uri>.
+   personal copy of another project as a subproject your home project.
+   Their names begin with <uri>&obsbranch1;</uri>.
+  </para>
+  <para>
+   There are different reasons to branch a package:
   </para>
-  <para>Branching a project has usually the following reasons:</para>
   <itemizedlist>
    <listitem>
     <para>
-     Allows you to make changes without affecting the original project.
+     Making changes without affecting the original project
     </para>
    </listitem>
    <listitem>
     <para>
-     Apply changes in your branch to try out a different path of development.
+     Applying changes in a branch to try out a different path of development
     </para>
    </listitem>
    <listitem>
     <para>
-     Modify the source code, build it, try it out, and submit back your
-     changes to the original project.
+     Modifying the source code, building it, trying the effect of the
+     changes, and submitting back changes to the original project.
```

Comment:
I would also point out here that this is the case when you don't have write access there. This is the way how you can contribute to any package you see in OBS.

Maybe this is is the most important point so it should be first?